src/main/java/com/greenfoxacademy/goddesstribesbackend/models/dtos/AuthenticationResponseDTO.java

```diff
@@ -1,8 +1,14 @@
 package com.greenfoxacademy.goddesstribesbackend.models.dtos;
 
+import io.swagger.annotations.ApiModelProperty;
+import io.swagger.annotations.SwaggerDefinition;
+
+@SwaggerDefinition
 public class AuthenticationResponseDTO {
 
+  @ApiModelProperty(position = 1)
   private Long userId;
+  @ApiModelProperty(position = 2)
```

Comment:
Is the order important?

Reply:
without it swagger list field in alfanumeric order